lib/ansiblelint/formatters/junitxml.py

```diff
@@ -0,0 +1,58 @@
+r"""JUnit XML formatter
+
+The XML Schema of JUnit XML format:
+
+- https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd
+  [http://windyroad.com.au/2011/02/07/apache-ant-junit-xml-schema/]
+"""
+from __future__ import absolute_import
+import datetime
+
+from .base import BaseFormatter
+from ..version import __version__ as VERSION
+
+
+_JUNIT_XML_TESTCASE = """\
+<testcase id="{rule_id}" name="{shortdesc}" classname="{filename}">
+        <failure type="ERROR" message="{message}">
+            rule_id={rule_id}
+            linenumber={linenumber}
+            line={line}
+        </failure>
+      </testcase>
+"""
+_JUNIT_XML = """\
```

Comment:
I think an actual XML generator is needed here

Reply:
(very sorry, I missed your review comment.)

Do you mean that XML template string should be built with element tree or something like that?
It should be better but I feel that a little too much because it's very small XML snippet.

Reply:
Are you sure that you properly escape **each** variable contents before interpolation, then?

Reply:
oh, i see. I understand what you mean. I'll check that further.